### PR TITLE
[risk=no] changing data structure from tree set to array list

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Method that adds missing bins to measurement concept results throws an error and many measurements do not show up in prod.
2. This was because tree set needs some sort of comparator which achilles result class do not have.
3. Using array list instead of tree set for now.

### CODE REVIEW

File: public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java [hunk 1]
```diff
@@ -1151,9 +1151,9 @@ else if(Integer.parseInt(ard.getStratum3()) == OTHER) {
             }
         } else {
 
-            TreeSet<AchillesResult> maleResults = new TreeSet<AchillesResult>();
-            TreeSet<AchillesResult> femaleResults = new TreeSet<AchillesResult>();
-            TreeSet<AchillesResult> otherResults = new TreeSet<AchillesResult>();
+            ArrayList<AchillesResult> maleResults = new ArrayList<AchillesResult>();
```
Comment (calbach) [inline public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java]: Better to use a general type if possible on the left-hand side, in this case `List`

Comment (calbach) [inline public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java]: In Java8 you can just exclude the generic type on the right-hand side `new ArrayList<>()`

**Review by calbach — COMMENTED** [review]
Is this fixing a bug? Was this a performance issue?

**Comment by calbach** [comment]
Oops - reread description, thanks. Sorry

**Comment by SrushtiGangireddy** [comment]
Yup, it was throwing error and not opening up measurement value graphs in prod.

**Review by calbach — APPROVED** [review]

**Pull request merged**